examples/animation/unchained.py

```diff
@@ -25,7 +25,7 @@
 # Generate random data
 data = np.random.uniform(0, 1, (64, 75))
 X = np.linspace(-1, 1, data.shape[-1])
-G = 1.5 * np.exp(-4 * X * X)
```

Comment:
It dates back to the early days of computers and compilers, when computers were slow and compilers were not so bright.  Multiplication is faster than taking a power, in general.  But now compilers recognize things like this and find the best algorithm.  In the case of numpy, I'm pretty sure there is explicit internal optimization of small integer powers so that we wouldn't have to use that ancient manual optimization.